misc/python/materialize/cli/ci_closed_issues_detect.py

```diff
@@ -0,0 +1,190 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+#
+# ci_closed_issues_detect.py - Detect references to already closed issues.
+
+import argparse
+import os
+import re
+import sys
+from dataclasses import dataclass
+
+import requests
+
+from materialize import buildkite, spawn
+
+ISSUE_RE = re.compile(r"\#([0-9]+) | materialize/issues/([0-9]+)", re.VERBOSE)
+
+FILE_ENDINGS = [".py", ".rs", ".yml", ".td", ".slt", ".md"]
+
+REFERENCE_RE = re.compile(
+    r"""
+    # This block contains expected references, don't report them
+    ^((?!.*
+    # Only track issues in Materialize repo
+    ( timely-dataflow\#
+    ))
+    .)*
+    # This block contains unexpected references to issues, report them
```

Comment:
Does this mean that they don't get picked up?

Reply:
No, it means they do get picked up, comment removed.